### ISSUE TITLE
Installing version 2.1 using pip requires Cython

### ISSUE THREAD

**Issue by takuyanakaoka** [rkern/line_profiler#109]
I noticed that `pip install line_profiler` requires cython from version 2.1.

```
$ pip list
pip (9.0.1)
setuptools (38.2.4)
wheel (0.30.0)
$ pip install line_profiler
Collecting line_profiler
  Using cached line_profiler-2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-oyNlhO/line-profiler/setup.py", line 22, in <module>
        alternatively use a release tarball from PyPI to build it without Cython.""")
    distutils.errors.DistutilsError: You need Cython to build the line_profiler from a git checkout, or
    alternatively use a release tarball from PyPI to build it without Cython.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-oyNlhO/line-profiler/
```

I'm using python 2.7.12 with pip 9.0.1. Is there any other requirements to install it using pip?
Or as in README, did you forget to add generated code in the archive?

Thanks!

**Comment by caethan** [comment]
Thanks for the report.  Yep, just did the 2.1 release today and forgot to include the generated code in the archive.  Will get that fixed ASAP.

**Comment by caethan** [comment]
Rolled back to 2.0 on PyPi while I get the correct build out

**Comment by caethan** [comment]
2.1.1 now released (and 2.1 removed) on PyPi, should fix the problem.  Please reopen if you see similar issues.  

Thanks again for the prompt report!  Helps a lot in getting these issues fixed before they cause problems for too many people.

**Comment by takuyanakaoka** [comment]
I confirmed that it works well now. Many thanks!
